src/webapp/components/app-bar/GlassAppBar.tsx

```diff
@@ -37,11 +39,23 @@ const useStyles = makeStyles(theme => ({
 export const GlassAppBar: React.FC = () => {
     const classes = useStyles();
 
-    const [country, setCountry] = React.useState(1);
+    const { currentUser } = useAppContext();
+    const { currentOrgUnitAccess, changeCurrentOrgUnitAccess } = useCurrentOrgUnitContext();
     const [action, setAction] = React.useState(1);
 
-    const changeCountry = (event: React.ChangeEvent<{ value: unknown }>) => {
-        setCountry(event.target.value as number);
+    const [orgUnit, setOrgUnit] = React.useState(currentOrgUnitAccess.orgUnitName);
```

Comment:
[naming] `orgUnitName`.

Reply:
Addressed in https://github.com/EyeSeeTea/glass-dev/pull/35/commits/c664c49dc8abcbeb0baeceac61d29b7d7275cb08